var/spack/repos/builtin/packages/py-non-regression-test-tools/package.py

```diff
@@ -18,8 +18,8 @@ class PyNonRegressionTestTools(PythonPackage):
 
     version("develop", branch="develop")
     version("main", branch="main")
+    version("1.1.6", tag="v1.1.6")
     version("1.1.4", tag="v1.1.4")
-    version("1.1.2", tag="v1.1.2")
```

Comment:
Is there a reason to remove this version rather than deprecate it (see docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions 
for guidance).

Reply:
(BTW: I do see that the tag no longer exists in the repository.)